src/components/action-buttons.component.tsx

```diff
@@ -55,14 +58,31 @@ const ActionButtons: React.FC<ActionButtonsProps> = ({
     medicationRequestStatus === MedicationRequestStatus.active &&
     mostRecentMedicationDispenseStatus !== MedicationDispenseStatus.declined;
 
+  const DesignName = () => {
```

Comment:
we could also move this logic in the place it's added to avoid the need for this naming.